app/scripts/background.js

```diff
@@ -41,8 +41,7 @@ import {
 } from './lib/enums'
 /* eslint-enable import/first */
 
-// METAMASK_TEST_CONFIG is used in e2e tests to set the default network to localhost
-const firstTimeState = { ...rawFirstTimeState, ...global.METAMASK_TEST_CONFIG }
+const firstTimeState = { ...rawFirstTimeState }
```

Comment:
I couldn't find a reference to `METAMASK_TEST_CONFIG`, so out it went.

Reply:
Yeah I think this might have been a remnant of an old e2e setup